containers/nginx-storage/delete-old-files.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash -e
+
+DIRECTORIES="$(echo /srv/storage/blender/{result/,'source/',verifier-output/})"
+
+for directory in ${DIRECTORIES[@]}; do
+    if [ ! -d "$directory" ]; then
+        echo "[crond][error][disk cleanup]: Please check if the \"$directory\" directory exist."
+        exit 1
+    fi
+done
+
+free_space="$(df -h | grep '/srv/storage' | awk '{ print $4; }' | tr -d 'G')"
+if [ -z "$free_space" ]; then
+    echo "[crond][error][disk cleanup]: There is a problem with checking free space on disk. Please check if mount point \"/srv/storage/\" exist."
+    exit 1
+fi
+
+find $DIRECTORIES      \
+    -mtime +30         \
+    -delete
+
+# Remove the part after the dot
+free_space=${free_space%.*}
+days=25
```

Comment:
If I am not wrong this will remove files older than 30 days. If it is not sufficient, then try 25,24,23...16. Why not check 29-26 as well ?